samples/domain_example.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) 2018 LabKey Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import unicode_literals
+
+from labkey.utils import create_server_context
+from labkey import domain
+
+labkey_server = 'localhost:8080'
+project_name = 'Study'  # Project folder name
+context_path = 'labkey'
+server_context = create_server_context(labkey_server, project_name, context_path, use_ssl=False)
+
+###################
+# Create a list domain
+###################
+list_domain_definition = {
+    'kind': 'IntList',  # or 'VarList'
```

Comment:
Should we have a page on .org at the API level which documents all of the domains which can be created with this api and what the domain kind optional params are.

Reply:
Yes, we should also have API tests (in java?) for all the ones that we state we support.

Reply:
Even better would be an API endpoint that would describe to users the available types and the associated optional parameters.